service.go

```diff
@@ -297,13 +297,29 @@ func (s *serviceImpl) Run(ctx context.Context) {
 	// since service.ExitFunc calls os.Exit(), we'll never get here
 }
 
-func (s *serviceImpl) AddRoute(name string, routes []string, methods []string, middlewares []Middleware, handler Handle) {
-	s.addRoute(s.publicRouter, publicSubsystem, name, routes, methods, middlewares, handler)
+func (s *serviceImpl) AddRoute(name string, routes []string, methods []string, middlewares []Middleware, metaFunc MetaFunc, handler Handle) {
+	s.addRouteWithMeta(s.publicRouter, publicSubsystem, name, routes, methods, middlewares, metaFunc, handler)
 }
 
 func (s *serviceImpl) addRoute(router *Router, subsystem, name string, routes []string, methods []string, middlewares []Middleware, handler Handle) {
+	defaultMetaFunc := func(_ *http.Request, _ RouterParams) map[string]string {
```

Comment:
could you, please, point to the place where metaFunc is not an empty map? 
Is it in PR code or it was added previously?

Reply:
It's not filled in this package anywhere, because the meta information comes from the package consumer. On line 73 it is exposed in the public `AddRoute` method. It is already in use by feeddownload. I commited the update by accident to master, but you can still review the commit if you want: https://bitbucket.org/xivart/travix.core.farecloud.feeddownload/commits/df5af0e252b1bc309f9ea4d302336e7c63bba69c

In the Kibana logs I saw that not all log messages contain the meta information, but at least some of them do. So expect another PR coming your way to fix it. 😄